docs/source/searching.rst

```diff
@@ -0,0 +1,16 @@
+.. _searching:
+
+Searching
+=========
+
+Searching is a process of finding items in already sorted list. This process can be implemented via many different algorithms. Following is the list of searching algorithm which are implemented in Pydsa
```

Comment:
* Searching doesn't require the list to be sorted.
* Line length is > than 79 chars
* Pydsa --> PyDSA